model-optimizer/extensions/back/StridedSliceMasksNormalizer.py

```diff
@@ -14,32 +14,23 @@
  limitations under the License.
 """
 
-from extensions.back.ConvolutionNormalizer import DeconvolutionNormalizer
-from extensions.back.CropToStridedSlice import CropToStridedSlice
 from mo.back.replacement import BackReplacementPattern
 from mo.front.common.partial_infer.utils import int64_array
-from mo.graph.graph import Graph, Node
+from mo.graph.graph import Graph
 
 
 class StridedSliceMasksNormalizer(BackReplacementPattern):
     enabled = True
     force_clean_up = True
 
     def run_after(self):
+        from extensions.back.ConvolutionNormalizer import DeconvolutionNormalizer
+        from extensions.back.CropToStridedSlice import CropToStridedSlice
         return [CropToStridedSlice, DeconvolutionNormalizer]
 
-    @staticmethod
-    def pattern():
-        return dict(
-            nodes=[
-                ('strided_slice', dict(type='StridedSlice'))
-            ],
-            edges=[]
-        )
-
-    def replace_pattern(self, graph: Graph, match: [str, Node]):
-        node = match['strided_slice']
-        assert node.has_valid('begin_mask')
-        assert node.has_valid('end_mask')
-        node.begin_mask = int64_array([1 - i for i in node.begin_mask])
-        node.end_mask = int64_array([1 - i for i in node.end_mask])
+    def find_and_replace_pattern(self, graph: Graph):
+        for node in graph.get_op_nodes(op='StridedSlice'):
```

Comment:
was matched by type. Could be a valuable if we have changes the node itself during some transformation -- not replacing it

Reply:
Changed to matching by `type` as it was before my changes